src/OpenTelemetry.Extensions.EventSource/README.md

```diff
@@ -17,9 +17,38 @@ dotnet add package OpenTelemetry.Extensions.EventSource --prerelease
 
 ## Usage Example
 
+### Configured using dependency injection
+
+```csharp
+IHost host = Host.CreateDefaultBuilder(args)
+    .ConfigureLogging(builder =>
+    {
+        builder.ClearProviders();
+
+        // Step 1: Configure OpenTelemetry logging...
+        builder.AddOpenTelemetry(options =>
+        {
+            options
+                .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService("MyService"))
```

Comment:
Maybe use the hot new `ConfigureResource` method here instead of the `SetResourceBuilder` method

Reply:
Updated